### PR TITLE
testcase: Adds new unit test case for resource alicloud_privatelink_vpc_endpoint alicloud_privatelink_vpc_endpoint_service alicloud_privatelink_vpc_endpoint_service_user

### PR DESCRIPTION
**Pull request by thisnihonglei**

### CODE REVIEW

File: alicloud/resource_alicloud_privatelink_vpc_endpoint_service_test.go [hunk 1]
```diff
@@ -196,3 +204,255 @@ var AlicloudPrivatelinkVpcEndpointServiceMap = map[string]string{
 func AlicloudPrivatelinkVpcEndpointServiceBasicDependence(name string) string {
 	return ""
 }
+
+func TestAccAlicloudPrivatelinkVpcEndpointService_unit(t *testing.T) {
```
Comment (thisnihonglei) [inline alicloud/resource_alicloud_privatelink_vpc_endpoint_service_test.go:208]: 单测覆盖率99.1%

File: alicloud/resource_alicloud_privatelink_vpc_endpoint_service_user_test.go [hunk 2]
```diff
@@ -80,3 +90,198 @@ func AlicloudPrivatelinkVpcEndpointServiceUserBasicDependence(name string) strin
 
 `, name)
 }
+
+func TestAccAlicloudPrivatelinkVpcEndpointServiceUser_unit(t *testing.T) {
```
Comment (thisnihonglei) [inline alicloud/resource_alicloud_privatelink_vpc_endpoint_service_user_test.go:94]: 单测覆盖率97.4%

File: alicloud/resource_alicloud_privatelink_vpc_endpoint_test.go [hunk 3]
```diff
@@ -234,3 +242,374 @@ func AlicloudPrivatelinkVpcEndpointBasicDependence(name string) string {
 	}
 `, name)
 }
+
+func TestAccAlicloudPrivatelinkVpcEndpoint_unit(t *testing.T) {
```
Comment (thisnihonglei) [inline alicloud/resource_alicloud_privatelink_vpc_endpoint_test.go:246]: 单测覆盖率96.2%

File: alicloud/resource_alicloud_privatelink_vpc_endpoint_service_test.go (hunk 1 above)
Comment (xiaozhu36) [reply alicloud/resource_alicloud_privatelink_vpc_endpoint_service_test.go:208]: 单元测试的name 都以 TestUnit 开头吧

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

**Pull request merged**